src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -1957,6 +1957,10 @@ void MessageHelper::SendSystemRequestNotification(
 
   content[strings::params][strings::connection_key] = connection_key;
 
+  policy::PolicyHandlerInterface& policy_handler = app_mngr.GetPolicyHandler();
+  content[strings::msg_params][strings::timeout] =
+      policy_handler.TimeoutExchangeSec();
+
```

Comment:
This parameter is only necessary when a URL is provided, #1496 contains the correct changes from what I can tell.